apps/src/applab/applab.js

```diff
@@ -1089,6 +1090,15 @@ Applab.onReportComplete = function(response) {
   displayFeedback();
 };
 
+/**
+ * Generates a library from the functions in the project code
+ */
+function getLibrary() {
+  var temporaryInterpreter = new JSInterpreter({studioApp: studioApp()});
+  temporaryInterpreter.parse({code: studioApp().getCode()});
+  return temporaryInterpreter.getFunctionsAndParams(studioApp().getCode());
```

Comment:
Is there any teardown required for the JSInterpreter?

Reply:
Nothing that's causing problems yet. Since this is a pilot. I'll leave this as-is and re-investigate if we end up going forward with this.

Reply:
👍